src/components/AddNewTodoForm.tsx

```diff
@@ -124,13 +136,16 @@ export const AddNewTodoForm: FC<Props> = () => {
                 />
               </Categories>
               <SelectMenu id="formSelectMenu">
-                <StyledOption value={0} text={i18n.t("modal:howImportant")} />
-                <StyledOption value={1} text={i18n.t("modal:veryImportant")} />
+                <StyledOption value="low" text={i18n.t("modal:howImportant")} />
+                <StyledOption
+                  value="high"
+                  text={i18n.t("modal:veryImportant")}
```

Comment:
this translation isn't related to modal so key should be smth like 'addTodo:veryImportant'